packages/purpose-process/src/services/purposeService.ts

```diff
@@ -263,6 +283,46 @@ export function purposeServiceBuilder(
       });
       await repository.createEvent(event);
     },
+    async updatePurpose({
+      purposeId,
+      purposeUpdateContent,
+      organizationId,
+      correlationId,
+    }: {
+      purposeId: PurposeId;
+      purposeUpdateContent: ApiPurposeUpdateContent;
+      organizationId: TenantId;
+      correlationId: string;
+    }): Promise<{ purpose: Purpose; isRiskAnalysisValid: boolean }> {
+      logger.info(`Updating Purpose ${purposeId}`);
+      return await updatePurposeInternal(
+        purposeId,
+        purposeUpdateContent,
+        organizationId,
+        "Deliver",
```

Comment:
Could you replace `"Deliver"` with `eserviceMode.deliver`? (Here and in the other parts where it occurs)